NEWS.md

```diff
@@ -367,6 +367,13 @@ Deprecated or removed
 
   * Instead of `linrange`, use `linspace` ([#9666]).
 
+  * The functions `is_valid_char`, `is_valid_ascii`, `is_valid_utf8`, `is_valid_utf16`, and
+    `is_valid_utf32` have been replaced by generic `isvalid` methods.
+    The single argument form `isvalid(value)` can now be used for values of type `Char`, `ASCIIString`,
+    `UTF8String`, `UTF16String` and `UTF32String`.
+    The two argument form `isvalid(type, value)` can be used with the above types, with values
+    of type `Vector{UInt8}`, `Vector{UInt16}`, `Vector{UInt32}`, and `Vector{Char}`.
```

Comment:
Ah, thanks, didn't know about that... (learning as fast as I can!), all done now.